config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "https://www.botkube.io/"
```

Comment:
Please also change line `For complete documentation visit www.botkube.io` to `For complete documentation, visit [botkube.io](https://botkube.io) website` in the main  `README.md` omn root